pom.xml

```diff
@@ -185,8 +187,8 @@ flexible messaging model and an intuitive client API.</description>
     <mysql-jdbc.version>8.0.11</mysql-jdbc.version>
     <elasticsearch.version>6.3.2</elasticsearch.version>
     <presto.version>0.206</presto.version>
-    <flink.version>1.6.0</flink.version>
-    <scala.binary.version>2.11</scala.binary.version>
+    <flink.version>1.8.0</flink.version>
+    <scala.binary.version>2.12</scala.binary.version>
```

Comment:
any particular reason that you are changing scala version from 2.11 to 2.12? Changing scala version can potentially impacting the down stream application who is using flink. I would prefer leaving it as 2.11 for now unless there is a strong reason call out for changing this to 2.12.

Reply:
ok reverted the change